src/outfit.ts

```diff
@@ -93,6 +96,44 @@ export function freeFightOutfit(requirements: Requirement[] = []): void {
   if (haveEquipped($item`Snow Suit`) && get("snowsuit") !== "nose") cliExecute("snowsuit nose");
 }
 
+export function checkLatte(): boolean {
```

Comment:
I think "refreshlatte" may be a more apt name--I initially implemented this not to check current ingredients, but to check unlocked ingredients